setup.py

```diff
@@ -11,7 +11,7 @@
   'board_game' : ['pachi-py>=0.0.19'],
   'box2d': ['Box2D-kengz'],
   'classic_control': ['PyOpenGL'],
-  'mujoco': ['mujoco_py<1.0.0,>=0.4.3', 'imageio'],
+  'mujoco': ['mujoco_py>=1.5', 'imageio'],
```

Comment:
While this is gonna match the right thing anyways, I think this should be `mujoco-py>=1.50` since the version we jumped from 0.5.x -> 1.50.x.y.